### PR TITLE
draft data documentation

### PR DESCRIPTION
**Pull request by GerbrichFerdinands**
Added a page to the documentation for dataset drafts. 
I think it could be a bit more concise by moving all citations to the bottom. 
@J535D165 What do you think?

### CODE REVIEW

File: docs/source/datasets.rst [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+All papers in the dataset need to be screened, with the goal to select all relevant papers. 
+
+For testing and demonstrating ASReview, the software offers `three built-in datasets <#built-in-demonstration-datasets>`__
+It is also possible to perform an automated systematic review on `your own dataset <#using-your-own-data>`__.
+Every paper in the dataset carries metadata such as title, abstract, author, et cetera. 
+
+Built-in demonstration datasets
+---------------------
+The built-in datasets come from 'traditional' systematic reviews on various research topics. 
+All papers who were selected in the original systematic review are indicated.
```
Comment (J535D165) [inline docs/source/datasets.rst]: who -> which

File: docs/source/datasets.rst [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+
+Using your own data
+---------------------
+    This instruction is currently under construction. 
+    
+To carry out an automated systematic review on your own dataset, your datafile needs to adhere to a certain format. 
+
+ASReview accepts the following formats: 
+ - ``.ris``
```
Comment (J535D165) [inline docs/source/datasets.rst]: Please add a link to https://en.wikipedia.org/wiki/RIS_(file_format) and the full format:

Something like:

- **[Research Information Systems (RIS)](https://en.wikipedia.org/wiki/RIS_(file_format))**. Extension `.ris`
- **Tabular datasets**. Extension `.csv`, `.xlsx` & `.xls`. CSV files should be comma separated and UTF-8 encoded.

File: docs/source/datasets.rst [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+For testing and demonstrating ASReview, the software offers `three built-in datasets <#built-in-demonstration-datasets>`__
+It is also possible to perform an automated systematic review on `your own dataset <#using-your-own-data>`__.
+Every paper in the dataset carries metadata such as title, abstract, author, et cetera. 
+
+Built-in demonstration datasets
+---------------------
+The built-in datasets come from 'traditional' systematic reviews on various research topics. 
+All papers who were selected in the original systematic review are indicated.
+This information can be used to compare performance of ASReview with the original systematic review.
```
Comment (J535D165) [inline docs/source/datasets.rst]: performance -> the performance

File: docs/source/datasets.rst [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+To perform an automated systematic review, ASReview requires a dataset of papers.
+The dataset of papers usually results from an online library search for all studies related to the topic to review.
+All papers in the dataset need to be screened, with the goal to select all relevant papers. 
+
+For testing and demonstrating ASReview, the software offers `three built-in datasets <#built-in-demonstration-datasets>`__
+It is also possible to perform an automated systematic review on `your own dataset <#using-your-own-data>`__.
+Every paper in the dataset carries metadata such as title, abstract, author, et cetera. 
+
+Built-in demonstration datasets
+---------------------
```
Comment (J535D165) [inline docs/source/datasets.rst]: This one gives issues with rendering. The underline should equal (or exceed) the length of the title. Same length is preferred.

**Comment by J535D165** [comment]
Closes #112

File: docs/source/datasets.rst [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+
+**A systematic literature review on fault prediction performance in software engineering**
+T. Hall, S. Beecham, D. Bowes, D. Gray, S. Counsell, in IEEE Transactions on Software Engineering, vol. 38, no. 6, pp. 1276-1304, Nov.-Dec. 2012. https://doi.org/10.1109/TSE.2011.103
+
+Dataset publication https://zenodo.org/record/1162952#.XiBgMi2ZNQK 
+
+Call: ``example_hall``
+
+
+Cohen
```
Comment (qubixes) [inline docs/source/datasets.rst]: It should be noted that the one included is the "ACEInhibitors" data set (there are multiple).

File: docs/source/datasets.rst [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+
+Built-in demonstration datasets
+---------------------
+The built-in datasets come from 'traditional' systematic reviews on various research topics. 
+All papers who were selected in the original systematic review are indicated.
+This information can be used to compare performance of ASReview with the original systematic review. 
+
+Van de Schoot
+~~~~~~~~~~~~~~
+A dataset of 11,395 papers on posttraumatic stress disorder. Of these papers, 34 were included in the systematic review.
```
Comment (qubixes) [inline docs/source/datasets.rst]: Where did you get this number? This is probably the number of papers with duplicates? Either way, it's probably less confusing to use the number of papers that are actually in the 'example_ptsd' data set, which has the duplicates removed. It should be in the 57xx range I believe.

File: docs/source/datasets.rst [hunk 7]
```diff
@@ -0,0 +1,72 @@
+Datasets
+========================
+To perform an automated systematic review, ASReview requires a dataset of papers.
+The dataset of papers usually results from an online library search for all studies related to the topic to review.
```
Comment (qubixes) [inline docs/source/datasets.rst]: Perhaps:
"To create such a dataset for a systematic review, typically an online library search is performed for all studies related to a particular topic."
?

**Comment by GerbrichFerdinands** [comment]
Thanks for the useful comments! I will process them next week.

File: docs/source/datasets.rst [hunk 8]
```diff
@@ -0,0 +1,72 @@
+Datasets
+========================
+To perform an automated systematic review, ASReview requires a dataset of papers.
```
[the remark this answers is not in the document]
Comment (Rensvandeschoot) [reply docs/source/datasets.rst]: It's actually a data set containing titles and abstracts, not the paper itself. You can also have a file containing all the pdf files for the full text and that is not what is needed for us, right?

File: docs/source/datasets.rst [hunk 9]
```diff
@@ -0,0 +1,72 @@
...
+The dataset of papers usually results from an online library search for all studies related to the topic to review.
+All papers in the dataset need to be screened, with the goal to select all relevant papers. 
+
+For testing and demonstrating ASReview, the software offers `three built-in datasets <#built-in-demonstration-datasets>`__
```
[the remark this answers is not in the document]
Comment (Rensvandeschoot) [reply docs/source/datasets.rst]: papers -> titles+abstracts

File: docs/source/datasets.rst (hunk 1 above)
Comment (Rensvandeschoot) [reply docs/source/datasets.rst]: what do you mean with 'traditional'? I would maybe refer to PRISMA based reviews?

File: docs/source/datasets.rst [hunk 10]
```diff
@@ -0,0 +1,72 @@
...
+This information can be used to compare performance of ASReview with the original systematic review. 
+
+Van de Schoot
+~~~~~~~~~~~~~~
+A dataset of 11,395 papers on posttraumatic stress disorder. Of these papers, 34 were included in the systematic review.
+
+    "We performed a systematic search to identify longitudinal studies that applied LGMM, latent growth curve analysis, or hierarchical cluster analysis on symptoms of posttraumatic stress assessed after trauma exposure."
+
+**Bayesian PTSD-Trajectory Analysis with Informed Priors Based on a Systematic Literature Search and Expert Elicitation**
+Rens van de Schoot ORCID Icon, Marit Sijbrandij, Sarah Depaoli ORCID Icon, Sonja D. Winter ORCID Icon, Miranda Olff & Nancy E. van Loey
```
Comment (Rensvandeschoot) [inline docs/source/datasets.rst]: remove ORCID icon?

File: docs/source/datasets.rst (hunk 8 above)
[the remark this answers is not in the document]
Comment (GerbrichFerdinands) [reply docs/source/datasets.rst]: How about: 

ASReview requires a data set representing all papers obtained in a systematic search. 
Each entry in the dataset holds metadata on a paper. The most common metadata are ``id``, ``author``, ``date``, ``title``, ``keywords`` and ``abstract``.

[the remark this answers is not in the document]
Comment (qubixes) [reply docs/source/datasets.rst]: Sounds better to me. Maybe you can split the mandatory metadata (title + abstract) and optional metadata?

**Comment by GerbrichFerdinands** [comment]
Thank you for all your helpful feedback. 
I hope you are happy with my changes (and that I added the link in the index file correctly). 
If you have any more feedback, let me know!

**Comment by J535D165** [comment]
Thanks @GerbrichFerdinands. This is a great job. We are very happy with it!

Merge!

**Pull request merged**